package.json

```diff
@@ -85,13 +85,14 @@
         "mocha": "^5.0.4",
         "rimraf": "^2.5.4",
         "typescript": "^2.4.2",
-        "vscode": "^1.1.4"
+        "vscode": "^1.1.14"
```

Comment:
Why 4 to 14?

Reply:
Just updating to latest.

Reply:
Sorry reading on my phone. I thought this was the unibeautify vscode package itself.